python_benchmarks/test_rope.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+from nvfuser import FusionDefinition, DataType
+from .core import run_benchmark, clear_cuda_cache
+import torch
+
+
+# Mimic the Hugging Face implementation:
+# https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/modeling_llama.py#L216
+def rope_with_cat_fusion(
+    fd: FusionDefinition,
+    batch_size: int,
+    seq_len: int,
+    num_heads: int,
+    features_per_head: int,
+) -> None:
+    q = fd.define_tensor(
```

Comment:
FYI, you can also do `fd.from_pytorch(pytorch_tensor)`, which does require you to pass in each tensor args... But that also sorted out the stride_order thing for you, which admittedly doesn't apply here.

Reply:
I see -- That's probably why we haven't been annoyed enough by having to specify the contiguity.